about.html

```diff
@@ -0,0 +1,137 @@
+
+    <div class="popup" id="popup">
+        <img src="./images/cloasepopup.svg" alt="close button" class="closepopup" onclick="closepopup()">
+        <ul>
+            <li><a href="about.html">About</a></li> 
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#form">Join</a></li>
+            <li><a href="#contact">Contact us</a></li>
+        </ul>
+       
+    </div>
+
+    <a href="./index.html"><img src="./images/240957a3-789c-4e2b-9fc7-34a08b3e9f0c.png" class="crypto-image" alt="crypto-logo" ></a>
+    <nav class="navigation"> 
+       
+        <div class="nav-text">
+        <div class="about"> <a href="./about.html"> About</a></div>
+        <div class="home"> <a href="./index.html">Home</a></div>
+        <div class="join"><a href="#form">Join</a></div>
+        <div class="contact" > <a href="#contact">Contact us </a></div>
+        </div>
+    </nav>
+
+    <section class="intro-section">
+      <div class="crypto-text">"Blockchain crypto!"</div>
+      
+      <h1 class="about-heading">NEW CRYPTO COIN 2022 INITIAL COIN OFFERING</h1>
+      <hr class="speaker-line1" />
+      <p class="crypto-words">
+        The crypto coin brings together investors ,financial institutions
+        governments and regular individuals to all have an impact in improving
+        the infrastructure and and increasing the level of technology used
+        across all sectors of life.This will be seen through the development of
+        the ecosystem which will me carried out by Artificial intelligence that
+        determines where to invest and develop in order to bring the highest
+        returns for the project.
+      </p>
+      <p>
+        Please contact us per Email for any further questions about new crypto
+        coin! newcryptocoin1@gmail.com
+      </p>
+      <h2>2022.12.01(TUE)~30(FRI)</h2>
+      <div id="contact">@ www.fakenewcryptocoin.com</div>
+    </section>
+
+    <section class="ico-section">
+      <h3>CRYPTO COIN 2022 INITIAL COIN OFFERING</h3>
+      <hr class="speaker-line1" />
+      The logo of crypto was decided decided through the conference held in 2021
+      <img
+        src="./images/240957a3-789c-4e2b-9fc7-34a08b3e9f0c.png"
+        alt="ico-image."
+        class="ico-pic"
+      />
+    </section>
+
+    <section class="ico-section">
+      <h3>SEE THE PAST INITIAL COIN OFFERINGS</h3>
+      See the past Initial Coin Offerings which took place in 2020 and 2021.
+        
+      <form action="https://formspree.io/f/xbjbawkn" method="post" id="form">
+        <input type="text" name="Name" maxlength="30" placeholder="Name">
+
+        <div class="submit">
+        Please type/text your email here to receive authentication email to participate in the ICO.
+        <textarea name="e-mail" cols="40" rows="1" placeholder="E-mail"></textarea>
+        <button type="submit" value="Submit">Submit</button>
+      </div>
+    </form>
```

Comment:
✅  Very Nice. You have gone the extra mile implementing the call to action for the page 👍